src/System.Data.SqlClient/src/System/Data/SqlClient/TdsParserStateObject.cs

```diff
@@ -3646,7 +3644,7 @@ internal int WarningCount
             }
         }
 
-        protected abstract object EmptyReadPacket { get; }
+        protected virtual PacketHandle EmptyReadPacket => default;
```

Comment:
Why virtual?

Reply:
with later revisions this still needs to be abstract, so reverted to the original.